totest-manager.py

```diff
@@ -21,6 +22,7 @@
 import signal
 import time
 import yaml
+import pika
```

Comment:
Needs to be represented on the rpm package as a dependency (and needs to be provided on SLE12SP3, current OS of packagelist vm - currently ttm is down as the update got deployed)